Common++/header/MacAddress.h

```diff
@@ -50,9 +53,8 @@ namespace pcpp
 		 * If the string doesn't represent a valid MAC address, the constructor throws an exception.
 		 * @param[in] addr the string representing the MAC address in format "00:00:00:00:00:00"
 		 */
-		template<typename T, typename = typename std::enable_if<std::is_convertible<T, std::string>::value>::type>
-		MacAddress(const T& addr) : MacAddress(static_cast<std::string>(addr)) {};
-
+		template <typename T, typename = typename std::enable_if<std::is_convertible<T, std::string>::value>::type>
+		MacAddress(const T& addr) : MacAddress(static_cast<std::string>(addr)){};
```

Comment:
Here it keeps the empty curly braces `{}` in the same line, as opposed to `IpAddress.h` where it didn't. Any idea why?

Reply:
maybe because it's a template?

Reply:
I think I know why, probably due to `;`

Reply:
yeah, after I removed `;`, it becomes normal.
But for now I can do manually fixing, but after when we apply a lot of files, it's hard to pick these syntax issues one by one.

Reply:
oh, maybe we should just remove the `;`

Reply:
> yeah, after I removed `;`, it becomes normal. But for now I can do manually fixing, but after when we apply a lot of files, it's hard to pick these syntax issues one by one.

we should fix things whenever we see them, of course we might miss some...